src/syft/core/io/location/specific.py

```diff
@@ -28,8 +28,7 @@ def icon(self) -> str:
 
     @property
     def pprint(self) -> str:
-        output = f"{self.icon} {self.name} ({self.class_name})@{self.id.emoji()}"
-        return output
+        return f"{self.icon} {self.name} ({self.class_name})@{self.id.emoji()}"
```

Comment:
Q: Why this change?
I was thinking that the ```output = ...``` is there for debugging purposes (if you have ```pdb``` and then you want to check out the message with a simple ```print(output)```.